wazo_acceptance/steps/line.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from behave import then
+from behave import when
+
+from hamcrest import assert_that
+from hamcrest import has_entries
+from hamcrest import has_item
+
+
+@when(u'I update lines with infos')
+def when_i_update_lines_with_infos(context):
+    for row in context.table:
+        row = dict(zip(row.headings, row.cells))
```

Comment:
You can use `as_dict`. See https://github.com/wazo-pbx/wazo-acceptance/blob/master/wazo_acceptance/steps/user.py#L57

Reply:
Lol, it seems I haven't search enough :)

Reply:
Well it wasn't in the behave documentation ...